test/unit/annotation_layer_spec.js

```diff
@@ -172,4 +188,35 @@ describe('Annotation layer', function() {
       expect(borderStyle.verticalCornerRadius).toEqual(0);
     });
   });
+
+  describe('FileAttachmentAnnotation', function() {
+    var loadingTask;
+    var annotations;
+
+    beforeEach(function() {
+      var pdfUrl = combineUrl(window.location.href,
+                              '../pdfs/annotation-fileattachment.pdf');
+      loadingTask = PDFJS.getDocument(pdfUrl);
+      waitsForPromiseResolved(loadingTask.promise, function(pdfDocument) {
```

Comment:
One small suggestion, to make this code a bit simpler: Remember that you can chain promises, i.e. this could instead become:
```
waitsForPromiseResolved(loadingTask.promise, function (pdfDocument) {
  return pdfDocument.getPage(1).then(function (pdfPage) {
    return pdfPage.getAnnotations().then(function (pdfAnnotations) {
      annotations = pdfAnnotations;
    });
  });
});
```